lib/di.dart

```diff
@@ -515,10 +515,15 @@ Future setup(
         _nodeSource,
         type ?? getIt.get<AppStore>().wallet!.type,
         getIt.get<SettingsStore>(),
+        getIt.get<NodeListViewModel>()
     ));
 
-  getIt.registerFactory(
-      () => NodeCreateOrEditPage(getIt.get<NodeCreateOrEditViewModel>()));
+  getIt.registerFactoryParam<NodeCreateOrEditPage, List<dynamic>?, void>((List<dynamic>? args, _) {
+    final node = args?.first as Node?;
+    final isSelected = args?[1] as bool?;
```

Comment:
better to use a Map instead of a List for args, cuz for any reason, if the passed args were not of the expected length, this will throw an index out of range